crypto/ec/ecx_key.c

```diff
@@ -137,29 +137,29 @@ int ossl_ecx_compute_key(ECX_KEY *peer, ECX_KEY *priv, size_t keylen,
         if (OPENSSL_s390xcap_P.pcc[1]
                 & S390X_CAPBIT(S390X_SCALAR_MULTIPLY_X25519)) {
             if (s390x_x25519_mul(secret, peer->pubkey, priv->privkey) == 0) {
-                ERR_raise(ERR_LIB_PROV, PROV_R_FAILED_DURING_DERIVATION);
+                ERR_raise(ERR_LIB_PROV, ERR_LIB_EC);
                 return 0;
             }
         } else
 #endif
         if (ossl_x25519(secret, priv->privkey, peer->pubkey) == 0) {
-            ERR_raise(ERR_LIB_PROV, PROV_R_FAILED_DURING_DERIVATION);
-            return 0;
-        }
+                ERR_raise(ERR_LIB_PROV, ERR_LIB_EC);
+                return 0;
+            }
     } else {
 #ifdef S390X_EC_ASM
         if (OPENSSL_s390xcap_P.pcc[1]
                 & S390X_CAPBIT(S390X_SCALAR_MULTIPLY_X448)) {
             if (s390x_x448_mul(secret, peer->pubkey, priv->privkey) == 0) {
-                ERR_raise(ERR_LIB_PROV, PROV_R_FAILED_DURING_DERIVATION);
+                ERR_raise(ERR_LIB_PROV, ERR_LIB_EC);
                 return 0;
             }
         } else
 #endif
         if (ossl_x448(secret, priv->privkey, peer->pubkey) == 0) {
-            ERR_raise(ERR_LIB_PROV, PROV_R_FAILED_DURING_DERIVATION);
-            return 0;
-        }
+                ERR_raise(ERR_LIB_PROV, ERR_LIB_EC);
```

Comment:
and here is off again.

Reply:
For some reason the style check did detect issues here and this was the solution. I might check the style-checker for some errors

Reply:
Okay, I changed it back to normal